apps/web-giddh/src/app/shared/header/components/account-add-new-details/account-add-new-details.component.html

```diff
@@ -442,7 +442,7 @@ <h3>Address {{i+1}}</h3>
                 <div class="button-group">
                     <!-- <button class="btn btn-cancel">Cancel</button> -->
                     <button type="submit" class="btn btn-success" [ladda]="createAccountInProcess$ | async"
-                        [disabled]="addAccountForm.invalid">Save</button>
+                        [disabled]="addAccountForm.invalid || !(isGstValid$ | async)">Save</button>
```

Comment:
<div class="col-sm-4" *ngIf="gst.get('gstNumber').value && isGstValid"> line 144 refers to `isGstValid` either revert the observable change and keep using variable or handle this code on line 144